2-mandatory/2-water-bottle.js

```diff
@@ -17,20 +17,29 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function () {
+  fillUp () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100;
   },
-  pour: function () {
+  pour () {
     // calling this function should increase your bottle volume by 10 units;
+    if (this.volume != 100) {
+      this.volume += 10;
+    }
   },
-  drink: function () {
+  drink () {
     // calling this function should decrease your bottle volume by 10 units;
+    if (this.volume != 0) {
+      this.volume -= 10;
+    }
   },
-  isFull: function () {
+  isFull () {
     // this function should return true if your bottle is full;
+    return this.volume === 100;
```

Comment:
I would advise you to avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) and instead define the max capacity in a single property that you refer to instead.